### PR TITLE
11389 implements design changes and form for email update notifications

### PR DESCRIPTION
**Pull request by apisandipas**
## Description
Implements design changes and form for email notification signups.

Currently the form **DOES NOT** submit correctly. Please see original ticket for discussion regarding that.

closes department-of-veterans-affairs/va.gov-cms/issues/11389

## Testing done & Screenshots
Desktop Design
Screenshot 2022-11-30 at 1 04 46 PM

Mobile Design
Screenshot 2022-11-30 at 1 05 06 PM

## QA steps
- [ ] verify successful submission _with stakeholders_ (TBD how from @mmiddaugh ) 

## Acceptance criteria
Veterans and beneficiaries viewing the redesigned homepage can signup for email updates using a component at the bottom of the page which 
- [x] uses a stacked design in desktop and mobile
- [x] has intro text ""Sign up to get the latest VA updates"
- [x] has the label "Email address" above the field
- [x] has actionable "Sign up" button below the field. 

- [ ] verify successful submission _with stakeholders_ (TBD how from @mmiddaugh ) 
- [ ] approval/sign off from Laura Flannery 

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by apisandipas** [comment]
@laflannery This PR is ready for your review

**Comment by laflannery** [comment]
Confirmed the visual Email Address label looks good and works well with screen reader.

However, `label` is not a valid attribute for the`<input>` element.  Could that be removed? (I think this might have been the old `aria-label` which is now replaced by the visual label)
image

**Comment by jilladams** [comment]
@apisandipas are there Analytics events for the email button as well? I think those were in the other PR, and this branch is just behind, but could you confirm?

**Comment by apisandipas** [comment]
@jilladams This branch has had main merged in and now Google Analytics events are being tracked on the functioning email form. Additional update has been made as requested by @laflannery 

This PR is now ready for code review.

**Comment by jilladams** [comment]
GA event: `{event: "homepage-email-sign-up", action: "Homepage email sign up", gtm.uniqueEventId: 45}`

Approving.

**Review by jilladams — APPROVED** [review]

**Comment by mmiddaugh** [comment]
Feedback from @TiffanyPender during accessibility testing: 

> When navigating the form controls rotor, the fields related to signing up for VA updates are not descriptive enough. Recommend updating the aria-label attributes to reflect more descriptive names (similar to the Search form controls in the body content of the page)

**Comment by apisandipas** [comment]
@mmiddaugh @TiffanyPender What shall we add to the label to make it more clear ?

**Comment by laflannery** [comment]
@TiffanyPender just to be clear, the visual design of this email form was updated and we are no longer using an aria-label, we are using a visual label.  If we were going to update this visual label it should be something that works both visually and in the rotor, we would not want this label to be duplicative (i.e Email address for updates may be a bit more clear for the rotor but would that be duplicative visually?)

image

**Comment by TiffanyPender** [comment]
Thanks @laflannery for letting me know about the update. It seems that the [version of the new homepage design](https://staging.va.gov/new-home-page/) that I completed my audit for is a bit different, and used the `aria-label` attribute. Very similarly to the Search field/button next to Top Pages. 

I think visually, including the `<label>` is more helpful. Without the additional textual context in the `aria-label` for screen readers, there may be some confusion. What are your thoughts? Do you think the current label text will do the job?

**Comment by laflannery** [comment]
@TiffanyPender we made this update (to expose the label) based on collab cycle feedback from Brian, to help with voice command users. I feel as though there is enough context to help users understand this field and the purpose without over complicating the label with additional information and we can probably leave as is. Let me know you agree so we can be sure of a direction for Bryan here

File: src/site/layouts/home-preview.drupal.liquid [hunk 1]
```diff
@@ -5,84 +5,8 @@
...
-    <div class="last-updated vads-u-padding-x--1 large-screen:vads-u-padding-x--0">
-      <div class="vads-u-display--flex above-footer-elements-container">
-        <div class="vads-u-flex--1 vads-u-text-align--right">
-          <span class="vads-u-text-align--right">
-              {% include "src/site/includes/medallia-feedback-button.drupal.liquid" %}
-          </span>
-        </div>
-      </div>
-    </div>
-  </div>
```
Comment (ryguyk) [inline src/site/layouts/home-preview.drupal.liquid:23]: Did this get removed accidentally?

Comment (apisandipas) [reply src/site/layouts/home-preview.drupal.liquid:23]: Yes, I will add it back.

**Comment by TiffanyPender** [comment]
> @TiffanyPender we made this update (to expose the label) based on collab cycle feedback from Brian, to help with voice command users. I feel as though there is enough context to help users understand this field and the purpose without over complicating the label with additional information and we can probably leave as is. Let me know you agree so we can be sure of a direction for Bryan here

Sounds good! Thanks @laflannery

**Review by jtmst — APPROVED** [review]
looks good in review instance and code changes look good to me

**Pull request merged**